projects/transformers/requirements.txt

```diff
@@ -22,9 +22,9 @@
 # Run pip install -r requirements.txt to reproduce the currently tested environment
 # Last update: 3/29/2021
 
-git+git:https://github.com/huggingface/transformers@master#egg=transformers
-datasets>=1.5.0
+transformers>=4.4.2,<=4.5.0
```

Comment:
If we fix on 4.5, that comment on the trainer mixin applies - 4.5 has a create_lr_scheduler function, which can be overriden instead of create_optimizer_and_scheduler.

Reply:
You're right. I'll keep that in mind for whenever we update.